server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -95,11 +96,16 @@
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class SegmentReplicationIT extends SegmentReplicationBaseIT {
 
+    @Before
+    private void setup() {
+        internalCluster().startClusterManagerOnlyNode();
```

Comment:
This will break tests that are inheriting SegmentReplicationIT and create their own manager node.

Reply:
It shouldn't. It will just add one more Cluster Manager node to the cluster. `SegmentReplicationRemoteStoreIT` creates cluster manager node and tests are passing.

Reply:
Ah was thinking it would not reach quorum bc there are 2.